homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,31 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int first = 1, second = 1, third;
```

Comment:
One definition/ declaration per line.

You should always initialize the variable created on stack with some default value.